base/test.jl

```diff
@@ -505,7 +505,7 @@ By default the `@testset` macro will return the testset object itself, though
 this behavior can be customized in other testset types.
 """
 macro testset(args...)
-    length(args) > 0 || error("No arguments to @testset")
+    !isempty(args) || error("No arguments to @testset")
```

Comment:
This is a nasty double negative now, could have just been `isempty(args) &&` surely?